language_task.py

```diff
@@ -113,7 +114,8 @@ def forward(self, input,hidden):
 parser.add_argument('--bptt', type=int, default=150,
                     help='sequence length')
 parser.add_argument('--cuda', action='store_true', default=False, help='use cuda')
-parser.add_argument('--seed', type=int, default=400,
```

Comment:
It is already set to use args.seed (equivalent to using args.random_seed). Is there something else that I need to change or am I missing something here?

Reply:
Yeah you've removed seed as an argument, and replaced with random_seed. You've changed the variable appropriately in both the numpy seed and torch seed, just need to modify the torch cuda one as well. Otherwise, there's a namespace error when running with cuda.